web/src/components/RequestDetails.vue

```diff
@@ -0,0 +1,139 @@
+Copyright 2024 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+    <div>
+      <v-card-title>
+          Request Details
+          <v-snackbar timeout="5000" color="primary" location="top" height="55">
+          Request output is <strong>downloading in the background</strong>, please wait
+          <v-progress-circular color="white" indeterminate></v-progress-circular>
+          <template v-slot:activator="{ props: snackbar }">
+            <v-tooltip top text="Download request output">
+              <template v-slot:activator="{ props: tooltip }">
+                <v-btn icon="mdi-folder-arrow-down-outline" variant="text" v-bind="mergeProps(snackbar, tooltip)"
+                  @click="getRequestOutput(requestDetails.request_id)">
+                </v-btn>
+              </template>
+            </v-tooltip>
+          </template>
+        </v-snackbar>
+      </v-card-title>
+      <v-alert v-if="requestDetails.status === 'success'" type="success" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> completed successfully.
+      </v-alert>
+      <v-alert v-else-if="requestDetails.status === 'running'" type="info" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> has <strong>{{ requestDetails.task_count - requestDetails.successful_tasks - requestDetails.failed_tasks }}</strong> Tasks remaining.
+      </v-alert>
+      <v-alert v-else-if="requestDetails.status === 'completed_with_errors'" type="warning" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> completed with <strong>{{ requestDetails.failed_tasks }}</strong> failed Tasks.
+      </v-alert>
+      <v-alert v-else type="error" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> was not successful.
+      </v-alert>
+      <v-card>
+          <v-list density="compact" v-model:opened="openGroups">
+            <v-list-group value="ids">
+              <template v-slot:activator="{ props }">
+                <v-list-item v-bind="props" title="Associated IDs"></v-list-item>
+              </template>
+              <v-list-item title="Request ID:">
+                  {{ requestDetails.request_id }}
+              </v-list-item>
+              <v-list-item title="Evidence ID:">
+                  {{ requestDetails.evidence_id }}
+              </v-list-item>
+              <v-list-item title="Reason:">
+                  {{ requestDetails.reason }}
```

Comment:
What does this look like if the value for ```reason``` is null? Similar for some of the other values like requester. Do we want an empty space or have a default like "N/A" or "Null"